rules/repositories.bzl

```diff
@@ -68,6 +97,12 @@ def rules_ios_dependencies():
         sha256 = "5e82a98a591efda772a5ee96ed17bcad38338aafeba6055daab04a5d6c13ea50",
     )
 
+    rules_ios_cfg(
```

Comment:
If we want to pursue @dostrander 's loading phase `kwargs` injection we'll need something like this. The PR is in a WIP / draft state so we can move forward on that if needed.

Reply:
Open to a better suggestion to switch in the loading phase if someone has it. I don't know of a great way you can introspect a `StarlarkCallable` off the top of my head and this might be better anyways.